src/Compilers/CSharp/Test/IOperation/IOperation/IOperationTests_IForEachLoopStatement.cs

```diff
@@ -4844,21 +4844,13 @@ static class Extensions
     }
     .finally {R5}
     {
-        CaptureIds: [1]
         Block[B4] - Block
             Predecessors (0)
-            Statements (1)
-                IFlowCaptureOperation: 1 (OperationKind.FlowCapture, Type: null, IsImplicit) (Syntax: 'new Program()')
-                  Value: 
-                    IConversionOperation (TryCast: True, Unchecked) (OperationKind.Conversion, Type: System.IAsyncDisposable, IsImplicit) (Syntax: 'new Program()')
-                      Conversion: CommonConversion (Exists: True, IsIdentity: False, IsNumeric: False, IsReference: True, IsUserDefined: False) (MethodSymbol: null)
-                        (ImplicitReference)
-                      Operand: 
-                        IFlowCaptureReferenceOperation: 0 (OperationKind.FlowCaptureReference, Type: System.Collections.Generic.IAsyncEnumerator<System.String>, IsImplicit) (Syntax: 'new Program()')
+            Statements (0)
```

Comment:
>Statements (0) [](start = 12, length = 14)

Does the change here match the IL that we would emit?

Reply:
This is async, so the IL we emit looks nothing like this. This is however the correct representation of what we 'would' emit before lowering to the state machine happens.